packages/angular/src/migrations/update-15-2-0/update-typescript-target.spec.ts

```diff
@@ -1,3 +1,4 @@
+import { UnitTestRunner } from '@nrwl/angular/src/utils/test-runners';
```

Comment:
Should this import be relative?

Reply:
Absolutely. It was already there so I didn't notice (the change was from VSCode organizing imports feature). Fixed now.